cvat/apps/engine/views.py

```diff
@@ -475,6 +477,23 @@ def data(self, request, pk):
 
                 elif data_type == 'preview':
                     return sendfile(request, frame_provider.get_preview())
+
+                elif data_type == 'context_image':
+                    if db_task.dimension == DimensionType.DIM_3D:
```

Comment:
@azhavoro , @Marishka17 ,

Could you please give some ideas how to implement the functionality in accordance with our general strategy? I believe the information should follow the same way as images and video frames. Probably it should be packed together with PCD data.